usaspending_api/database_scripts/etl/location_delta_view.sql

```diff
@@ -11,11 +11,15 @@ WITH country_cte AS (
 			ELSE
 				pop_country_name
 		END AS location_string,
-		'country' AS location_type
+		'country' AS location_type,
+        COUNT(*) AS occurrences
```

Comment:
Do you mind indenting this to be at the same level as the rest of the SELECT?

Reply:
Can you actually fix some of the indentation elsewhere in this SQL?

Reply:
I'm doing some testing and might end up just closing this PR. While this change gives us more "realistic" results in _some_ cases, in other cases it completely messes up the "realistic" results.

For example, if we sort by `occurrences` and you search for `Denver` then it does put `Denver, Colorado` at the top as you would (most likely) expect. However, if you then search for `New York City` it puts `Long Island City, New York` at the top because this location occurs more often than `New York City, New York` (which is actually the 5th result when sorted by occurrences).

There's not a perfect solution and in this testing it seems like this may end up causing confusion more often than not.